cpp/cmake/thirdparty/get_raft.cmake

```diff
@@ -53,8 +55,8 @@ endfunction()
 # To use a different RAFT locally, set the CMake variable
 # CPM_raft_SOURCE=/path/to/local/raft
 find_and_configure_raft(VERSION    ${CUGRAPH_MIN_VERSION_raft}
-                        FORK       rapidsai
-                        PINNED_TAG branch-${CUGRAPH_BRANCH_VERSION_raft}
+                        FORK       cjnolet
+                        PINNED_TAG build_2206_cuco_distance_component
```

Comment:
I'm assuming these lines will change?

Reply:
Yeah those will be reverted once the raft side is merged